client/my-sites/people/invite-people/index.jsx

```diff
@@ -89,8 +93,18 @@ const InvitePeople = React.createClass( {
 			return value;
 		} );
 
+		const filteredErrors = pickBy( errors, ( error, key ) => {
```

Comment:
Could we not deal with an array of errors still? [ { }, { } ].
Then I think things would be simpler. Sorry I just don't quite understand.